MANIFEST.in

```diff
@@ -0,0 +1,11 @@
+include README.md
+include prompt.py
```

Comment:
what we also need to include is `bootstrap.py`, the `protocol.*.json` files, and `reencrypt_wallet.py`

Reply:
fixed